frontend/test/pages/BasePage.ts

```diff
@@ -1,7 +1,9 @@
-import { By, until, WebDriver, Locator } from 'selenium-webdriver'
+import { By, until, WebDriver, Locator } from 'selenium-webdriver';
 
 export class BasePage {
   protected readonly driver: WebDriver
+  readonly skipTutorialDataTestId = 'btn-skip-tutorial'
```

Comment:
I don't see this being used everywhere, could this be an omission?

Reply:
This was probably a merge conflict. Resolving
```suggestion
```